news/2020-11-11-new-featured-artist-fuki.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: Fuki"
+date: 2020-11-11 09:00:00 +0000
+---
+
+We're proud to welcome **Fuki** aboard as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/114/header.jpg)
+
+If you're a fan of Japanese metal or rock, you might already know **Fuki** from her time as a vocalist in bands like *LIGHT BRINGER*, *Unlucky Morpheus* and *DOLL$BOXX*. She's participated in a wide range of projects, ranging from things like radio voice-overs, voice acting, narration. She has also sung the ending themes of anime like *Kaitou Joker*, *W'z* and even the theme song for *IKEMEN REVOLUTION - Love & Magic in Wonderland*.
+
+**11** new tracks from her are now available straight from one of her most recent albums titled *Million Scarlets*, including a track featured in the osu! World Cup 2019 Finals week, and a brand new showcase set from some of the minds of the Mappers' Guild.
```

Comment:
> one of her most recent albums

i think this is their only solo album?

Reply:
also the "a brand new showcase set" here should instead be "two brand new showcase sets" i suppose...

Reply:
fixed both